addons/mail/static/src/core/public_web/thread_model_patch.js

```diff
@@ -38,7 +38,7 @@ patch(Thread.prototype, {
                         !this.store.discuss.isActive &&
                         this.store.chatHub.opened.length < this.store.chatHub.maxOpened
                     ) {
-                        chatWindow.open();
+                        chatWindow.open({ shouldFocus: false });
```

Comment:
But even if no other chat window is opened, is there no autofocus when a new message is received?

Reply:
Good point :)